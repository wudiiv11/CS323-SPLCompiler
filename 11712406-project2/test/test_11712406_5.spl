int main() {
    int a, b, c;
    int d =(a = (b = (c = 1)));    
    return 1.2;
}