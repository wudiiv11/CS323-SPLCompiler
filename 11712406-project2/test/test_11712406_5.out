Error type 8 at Line 4: incompatible return type